pna_control/pna_control.py

```diff
@@ -99,8 +139,12 @@ def get_data(centerf: float,
             ifband: float = 5, 
             points: int = 201, 
             outputfile: str = "results.csv",
-            instr_addr : str = 'GPIB::16::INSTR',
-            sparam : str = 'S12'):
+            # instr_addr : str = 'GPIB::16::INSTR',
+            # instr_addr : str = 'TCPIP0::69.254.35.52::islip0::INSTR1',
```

Comment:
Do we need to keep the commented out instruments? If so, can you add comments describing each one?